packages/experience-docs/source/components/Icon/index.js

```diff
@@ -14,12 +14,15 @@ const getIcon = (icon) => find(ICONS, ({title}) => title === startCase(icon))
 const Icon = ({
   name,
   icon,
-  kind = 'flat'
+  kind = 'flat',
+  ...props
 }) => {
   const {src, title} = getIcon(icon)
-  return src && <SVGIcon kind={kind} svg={src} title={title} alt={title} />
+  return src && <SVGIcon {...props} kind={kind} svg={src} title={title} alt={title} />
```

Comment:
I'd really rather we avoid ever doing this. This basically says "pass anything to me and I'll pass it on". You lose all visibility of how this is being used and what you can expect to be passed to child components, so you can get unpredictable behaviour. If a component renders a child that requires a prop, that component must also require that prop explicitly.